src/lkl/setup.c

```diff
@@ -1110,6 +1013,10 @@ static void init_random()
     }
 
     fd = open("/dev/random", O_RDONLY);
+    /* Define ioctl() rather than including <sys/ioctl.h> to work around
+     * duplicate definitions of the _IO* family of macros.
+     */
+    extern int ioctl (int, int, ...);
```

Comment:
How will this change after re-layering, when this code moves to userspace?

Reply:
This suppression will go away because we will no longer be mixing kernel and user headers. This warnings existed before these changes